plugin/adsk/scripts/AEmayaUsdProxyShapeTemplate.mel

```diff
@@ -1,7 +1,66 @@
+global proc AEmayaUsdProxyShapeInfoNew(string $input)
+{
+    textFieldGrp -label "Root Layer" -editable 0
+                 -ann "Identifies the root layer of a stage. If a file path is shown in this field, the root layer is a file on disk. If a layerName is shown in this field, the root layer is an anonymous layer."
+                 mayaUsdProxyShapeRootLayer;
+
+    textFieldGrp -label "Default Prim" -editable 0
+                 -ann "As part of its metadata, each stage can identify a default prim. This is the primitive that is referenced in if you reference in a file."
+                 mayaUsdProxyShapeDefaultPrim;
+
+    AEmayaUsdProxyShapeInfoReplace($input);
+}
+
+global proc AEmayaUsdProxyShapeInfoReplace(string $input)
+{
+    // From the input attribute we simply want the node name.
+    string $nodeName = plugNode($input);
+    string $res[] = `ls -l $nodeName`;
+    string $fullNodeName = $res[0];
+
+    // Call python helpers to get the info we want to display.
+    string $rootLayer   = `python("import AETemplateHelpers; AETemplateHelpers.GetRootLayerName('" + $fullNodeName + "')")`;
+    string $defaultPrim = `python("import AETemplateHelpers; AETemplateHelpers.GetDefaultPrimName('" + $fullNodeName + "')")`;
+
+    textFieldGrp -edit -text $rootLayer mayaUsdProxyShapeRootLayer;
+    textFieldGrp -edit -text $defaultPrim mayaUsdProxyShapeDefaultPrim;
```

Comment:
This will update the two fields with the values from the current proxy shape being shown.